documentation/publish/package-your-extension.md

```diff
@@ -145,7 +145,7 @@ The most convenient way to package your extension is to use [web-ext build](/doc
 
 {% capture note %}
 
-See [http://www.info-zip.org/mans/zip.html](http://www.info-zip.org/mans/zip.html)
+See [https://linux.die.net/man/1/zip](https://linux.die.net/man/1/zip)
```

Comment:
Some copy for the link might make it a bit clearer what it's linking to e.g something like:

```markdown
See [the documentation for the `zip` command](https://linux.die.net/man/1/zip) for more information.
```

Renders as:

See [the documentation for the `zip` command](https://linux.die.net/man/1/zip) for more information.